src/test/ui/consts/issue-66342.rs

```diff
@@ -0,0 +1,9 @@
+// check-pass
+
```

Comment:
Can you add a comment explaining what this is testing (same with the other test)?